reference/customers.v2.yml

```diff
@@ -1289,6 +1289,60 @@ components:
       description: When creating a customer group category discount using the API it defaults to "products in this category and its subcategories". In the [store control panel](https://support.bigcommerce.com/s/article/Customer-Groups#pricing), this can be changed to either "products in this category only" or "products in this category and its subcategories". There are currently no settings to change this behavior with the API.
       x-internal: false
+    customerGroup_Update:
+      title: customerGroup_Update
+      type: object
+      properties:
+        name:
+          type: string
+          description: Name of the group.
+          example: Wholesale
+        is_default:
+          type: boolean
+          description: Determines whether new customers are assigned to this group by default.
+          example: false
+        category_access:
+          $ref: '#/components/schemas/categoryAccessLevel_Full'
+        discount_rules:
+          type: array
+          description: A collection of discount rules that are automatically applied to customers who are members of the group.
+          items:
+            type: object
+            properties:
+              type:
+                type: string
+                enum:
+                  - price_list
+                  - all
+                  - category
+                  - product
+              method:
+                type: string
+                enum:
+                  - percent
+                  - fixed
+                  - price
+              amount:
+                type: string
+                description: A float that specifies the value applied to the price modified. (Float, Float as String, Integer)
+                example: '"5.0000"'
+              price_list_id:
+                type: integer
+                description: If a customer group is assigned to a price list,`method` and `amount` are not shown. `type` and `price_list_id` are returned.
+                example: 3
+        date_created:
+          type: string
+          description: Date on which the customer group was created.
+          example: 2023-07-17 06:30:41
+        date_modified:
+          type: string
+          description: Date on which the customer group was last modified.
+          example: 2023-07-25 01:15:19
+        is_group_for_guests:
+          type: boolean
+          description: Describes whether the group is for guests. There can only be one customer group for guests at a time.
+      description: When updating a customer group category discount using the API it defaults to "products in this category and its subcategories". In the [store control panel](https://support.bigcommerce.com/s/article/Customer-Groups#pricing), this can be changed to either "products in this category only" or "products in this category and its subcategories". There are currently no settings to change this behavior with the API.
```

Comment:
```suggestion
      description: When updating a customer group category discount using the API, it defaults to "products in this category and its subcategories". In the [store control panel](https://support.bigcommerce.com/s/article/Customer-Groups#pricing), this can be changed to either "products in this category only" or "products in this category and its subcategories." There are currently no settings to change this behavior with the API.
```